addons/addon-base-raas/packages/base-raas-services/lib/study/__tests__/study-service.test.js

```diff
@@ -17,6 +17,7 @@ const _ = require('lodash');
 const ServicesContainer = require('@aws-ee/base-services-container/lib/services-container');
 const JsonSchemaValidationService = require('@aws-ee/base-services/lib/json-schema-validation-service');
 const AwsService = require('@aws-ee/base-services/lib/aws/aws-service');
+const { getSystemRequestContext } = require('@aws-ee/base-services/lib/helpers/system-context');
```

Comment:
Nit: Is system context really necessary in a unit test? Can't we just mock the authorization logic and move on with the test?

Reply:
For open data create/update, study-service uses `isSystem` to check authorization, which was an imported function. It's not quite straightforward to mock compared to a in class `isAuthorized` function.